templates/connect-inject-deployment.yaml

```diff
@@ -90,6 +90,9 @@ spec:
                 -release-name="{{ .Release.Name }}" \
                 -release-namespace="{{ .Release.Namespace }}" \
                 -listen=:8080 \
+                {{- if .Values.connectInject.transparentProxy.defaultEnabled }}
+                -enable-transparent-proxy \
```

Comment:
I'm just noticing this now and it's not a blocker for the release, but should this flag be renamed -default-enable-transparent-proxy in consul-k8s after the release? That's how the metrics flag defaults are named.

Reply:
That's a good catch! I'll a task to rename.